src/__test__/index.test.ts

```diff
@@ -58,93 +60,84 @@ const createPerson = (): Person => {
 }
 
 describe('Client', function() {
-  let store: NewStoreReply.AsObject
-  describe('.newStore', () => {
-    it('response should be defined and have an id', async () => {
-      store = await client.newStore()
-      expect(store).to.not.be.undefined
-      expect(store).to.haveOwnProperty('id')
-      expect(store.id).to.not.be.undefined
+  const bytes = Buffer.concat([Buffer.from(encode(0x01)), Buffer.from(encode(0x55)), randomBytes(32)])
+  const dbID = multibase.encode('base32', bytes).toString()
```

Comment:
@carsonfarmer it might make sense to pull ThreadId in from `core` here, but I just used this shortcut for now. Perhaps leaving that option up to the developer, but a recommended route.

Reply:
Sure that seems fine for now. @textile/threads-core is super lightweight, so recommending to use it (but not requiring it here) is probably fine.